lib/puppet/provider/package/dpkg.rb

```diff
@@ -162,4 +164,12 @@ def unhold
     end
   end
 
+  def is_package_installed?(name)
```

Comment:
A better name for the method would be just `package_installed?`. Also, since @resource is an instance variable, there is no need to pass `resource[:name]` as a parameter.